aspnet/fundamentals/owin.rst

```diff
@@ -1,8 +1,311 @@
-.. include:: /../common/stub-topic.txt
+OWIN
+====
 
-|stub-icon| OWIN
-================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+ASP.NET 5 supports OWIN, the Open Web Interface for .NET, which allows web applications to be decoupled from web servers. In addition, OWIN defines a standard way for middleware to be used in a pipeline to handle individual requests and associated responses. ASP.NET 5 applications and middleware can interoperate with OWIN-based applications, servers, and middleware.
 
-.. _issue: https://github.com/aspnet/Docs/issues/71
\ No newline at end of file
+In this article:
+	- `Running OWIN middleware in the ASP.NET pipeline`_
+	- `Using ASP.NET Hosting on an OWIN-based server`_
+	- `Run ASP.NET 5 on an OWIN-based server and use its WebSockets support`_
```

Comment:
I thought that we don't encourage people to use ASP.NET 5 on OWIN, but that it's fine to use OWIN middleware on ASP.NET 5.